api/nakadi-event-bus-api.yaml

```diff
@@ -333,7 +677,8 @@ paths:
         - name: batch_flush_timeout
           in: query
           description: |
-            Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached before this
+            Maximum time in seconds to wait for the flushing of each chunk; if the `batch_limit` is reached
+            before this
             time is reached the messages are immediately flushed to the client.
             A value of 0 will flush every batch events immediatly as soon as they come in.
```

Comment:
@valgog then it will be not consistent
With current behavior batch_limit is by default always 1.
With your suggestion it will be by default 1 only when batch_flush_timeout is also not defined